pkg/kubelet/kubelet_network.go

```diff
@@ -95,10 +95,6 @@ func omitDuplicates(kl *Kubelet, pod *v1.Pod, combinedSearch []string) []string
 		if _, exists := uniqueDomains[dnsDomain]; !exists {
 			combinedSearch[len(uniqueDomains)] = dnsDomain
 			uniqueDomains[dnsDomain] = true
-		} else {
-			log := fmt.Sprintf("Found and omitted duplicated dns domain in host search line: '%s' during merging with cluster dns domains", dnsDomain)
-			kl.recorder.Event(pod, v1.EventTypeWarning, "DNSSearchForming", log)
-			glog.Error(log)
```

Comment:
What are the possible error states that deduplication could indicate? Is this worth a V(3) for debugging?